docs/data-sources/dli_spark_templates.md

```diff
@@ -0,0 +1,143 @@
+---
+subcategory: "Data Lake Insight (DLI)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_dli_spark_templates"
+description: |-
+  Use this data source to get the list of the DLI spark templates.
+---
+
+# huaweicloud_dli_spark_templates
+
+Use this data source to get the list of the DLI spark templates.
+
+## Example Usage
+
+```hcl
+variable "template_id" {}
+
+data "huaweicloud_dli_spark_templates" "test" {
+  template_id = var.template_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the resource.
+  If omitted, the provider-level region will be used.
+
+* `template_id` - (Optional, String) Specifies the ID of the spark template to be queried.
+
+* `name` - (Optional, String) Specifies the name of the spark template to be queried.
+
+* `group` - (Optional, String) Specifies the group name to which the spark templates belong.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The data source ID.
+
+* `templates` - All templates that match the filter parameters.
+
+  The [templates](#templates_struct) structure is documented below.
+
+<a name="templates_struct"></a>
+The `templates` block supports:
+
+* `description` - The description of template.
+
+* `id` - The ID of template.
+
+* `name` - The name of template.
+
+* `body` - The body of template.
+
+  The [body](#templates_body_struct) structure is documented below.
+
+* `group` - The group of template.
+
+<a name="templates_body_struct"></a>
+The `body` block supports:
+
+* `app_parameters` - The input parameters of the main class, that is application parameters.
+
+* `name` - The Spark job name.
```

Comment:
Change word lowercase. Check the other parts.

Reply:
done